scripts/test_images.sh

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bash
+
+# Exit on error
+set -e
+
+# Consider command as failed when any component of the pipe fails:
+# https://stackoverflow.com/questions/1221833/pipe-output-and-capture-exit-status-in-bash
+set -o pipefail
+
+source ./scripts/test_lib.sh
+source ./scripts/build_lib.sh
+
+# Can't proceed without docker
+if ! command -v docker >/dev/null; then
+    log_error "cannot find docker"
+    exit 1
+fi
+
+# You can't run darwin binaries in linux containers
+if [[ $(go env GOOS) == "darwin" ]]; then
+    echo "Please use linux machine for release builds."
+    exit 1
+fi
+
+# Docker build
+ARCH=$(go env GOARCH)
+DOCKERFILE="Dockerfile-release.${ARCH}"
+BINARYDIR=${BINARYDIR:-"bin"}
+IMAGE=${IMAGE:-"test-etcd-image-deadbeaf1"}
+VERSION=${VERSION:-"default"}
+TAG="${IMAGE}:${VERSION}"
+
+# ETCD related values
+RUN_NAME="test_etcd"
+KEY="foo"
+VALUE="bar"
+
+# Build if binaries are not present
+if [ -z "$BINARYDIR" ]; then
+    run ./scripts/build.sh
+fi
+
+# Build a local image from bin directory
+if ! docker build -t "${TAG}" -f "${DOCKERFILE}" "${BINARYDIR}"; then
+    echo "Docker build unsuccessful. Exit code $?"
+    exit 1
+fi
```

Comment:
Why do you build the image? Can you use the images built by previous step directly? 

We can assume that the images are already available, how about passing in the image tag, i.e. v3.5.99?